src/Lean/Elab/Term/TermElabM.lean

```diff
@@ -313,6 +318,11 @@ structure Context where
 abbrev TermElabM := ReaderT Context $ StateRefT State MetaM
 abbrev TermElab  := Syntax → Option Expr → TermElabM Expr
 
+def Context.autoBoundImplicit (ctx : Context) : Bool :=
+  match ctx.autoBoundImplicitContext with
+    | .none => false
+    | .some subCtx => subCtx.flag
```

Comment:
Is this the `Context.autoImplicit` function that's mentioned as `Lean.Elab.Term.Context.autoImplicit` in the docstring of the `flag` field? If so, it should be actually deprecated and the docstring updated. If not, then it would be nice to find a way to clear up this confusion.

Reply:
Adding a deprecation message for this function - this is only here to avoid breaking something downstream anyway